extensions-contrib/compressed-bigdecimal/src/main/java/org/apache/druid/compressedbigdecimal/CompressedBigDecimalAggregatorFactory.java

```diff
@@ -65,19 +69,23 @@
    * @param fieldName fieldName metric field name
    * @param size      size of the int array used for calculations
    * @param scale     scale of the number
+   * @param strictNumberParsing if true, failure to parse strings to numbers throws an exception. otherwise 0 is
+   *                            returned
    */
   @JsonCreator
   public CompressedBigDecimalAggregatorFactory(
       @JsonProperty("name") String name,
       @JsonProperty("fieldName") String fieldName,
       @JsonProperty(value = "size", required = false) Integer size,
-      @JsonProperty(value = "scale", required = false) Integer scale
+      @JsonProperty(value = "scale", required = false) Integer scale,
+      @JsonProperty(value = "strictNumberParsing", required = false) Boolean strictNumberParsing
   )
   {
     this.name = name;
     this.fieldName = fieldName;
     this.size = size == null ? DEFAULT_SIZE : size;
     this.scale = scale == null ? DEFAULT_SCALE : scale;
+    this.strictNumberParsing = strictNumberParsing != null && strictNumberParsing;
```

Comment:
hmm, everything else has defaults set in the SqlAggregator. The only reason I double-checked it here was because the JsonCreator requires Objects

the scale and size are also both nullable and I could pass in null for those. it seems like I should be consistent with the parameters. Other SqlAggregators do add defaults, so do it in both locations or just the constructor?